### ISSUE TITLE
stream test fails (due to bug in test)

### ISSUE THREAD

**Issue by markstos** [node-strava/node-strava-v3#117]
This test is failing:

```
streams_test
    #activity()
           1) should return raw data associated to activity
```

URL called is like: `    url: 'routes/2849314360760640500/streams/?resolution=low',`

Error is: `StatusCodeError: 404 - {"message":"Resource Not Found","errors":[]}`

From reading the API docs, it seems like routes like this should exist, so maybe it's a bug in our test.

<Looks>

Yep. The test contains a hardcoded activity ID, which could have only possibly worked for the original developer.

The fix would be to generate the test activity-with-segment, then run the test, then remove it.

**Comment by markstos** [comment]
Good idea. I'm making you an owner of the new `node-strava` organization so you can implement it.

BTW, Strava seems not to mind if extra test accounts are created for the purpose of creating additional API keys. They are aware that my company creates them on behalf of customers so that each customer's domain has a separate API key. 

Creating a test account that's used exclusively test a Node.js Strava API client seems like a fair use.

**Comment by markstos** [comment]
@Kyle-Ski Is this test fixed by #162 ?

**Comment by Kyle-Ski** [comment]
I haven't seen this test failing when I've been working on it. And I just uncommented these lines and ran the tests with no failures there. I'm guessing I should leave these uncommented in #162 ?

```js
describe('streams_test', function () {
  before(function (done) {
    this.timeout(5000)

    testHelper.getSampleActivity(function (err, payload) {
      if (err) { return done(err) }

      _sampleActivity = payload

      _activity_id = _sampleActivity.id
      // _segmentEffort_id = _sampleActivity.segment_efforts[0].id;
      // _segment_id = _sampleActivity.segment_efforts[0].segment.id;

**Comment by markstos** [comment]
Sounds right. Closing this issue since the tests pass.
